core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/definitions/LdapServicesDecoder.scala

```diff
@@ -222,6 +229,19 @@ object LdapServicesDecoder {
       }
   }
 
+
+  private lazy val circuitBreakerDecoder: Decoder[CircuitBreakerConfig] =
+    Decoder
+      .instance { c =>
+        val circuitBreaker = c.downField("circuit_breaker")
+        if (circuitBreaker.failed) {
+          Right(DEFAULT_CIRCUIT_BREAKER_CONFIG)
+        } else {
+          val decoder = Decoder.forProduct2("max_retries", "reset_duration")(CircuitBreakerConfig.apply)
```

Comment:
do we have a test which tests what message will be seen by a user when he tries to set sth like this:

```
circuit_breaker:
  max_retries: 2
```

I wonder if this method of decoding will allow us to return meaningful error message? 
(Circe is pretty dump according to self explanatory decoding errors. We created own mechnism which makes it more user friendly, but sadly the cost is less pleasant decoding code.

_(We can do it in a second part of the task)_